packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -129,6 +131,15 @@ const getDocs: () => JSX.Element = () => {
         button will cycle through the different camera's in the users device. Typical usage is to enable this button on
         mobile devices. This button is enabled through the use of the `showCameraSwitcherInLocalPreview` prop.
       </Description>
+      <StoryBookBanner
+        // eslint-disable-next-line react/no-children-prop
+        palette={yellowBannerPalette}
+      >
+        <p style={{ display: 'inline-block' }}>
```

Comment:
use fluent components over raw html elements
```suggestion
        <Text>
```